### ISSUE TITLE
Launchpad Release

### ISSUE THREAD

**Issue by ethanluoyc** [google-deepmind/launchpad#41]
Hi Launchpad Team,

Thanks for trying to help us with the https://github.com/deepmind/launchpad/issues/32, however, I don't think we have a release for the new TF version. Both the nightly and stable versions?

I am currently using a wheel built from a source as mentioned in https://github.com/deepmind/launchpad/issues/30#issuecomment-1597262369. There are some additional fixes that seem to be required to work with TF 2.12.0. I am happy to create a PR if that is preferred (but maybe it's easier to update from internal). There seem to be some upgrades needed in terms of how to handle absl Status and tsl Status otherwise things are straightforward.

The update is quite useful as it is currently blocking using JAX agents in dm-acme at HEAD. dm-acme@HEAD now uses jax>0.4.3 which is incompatible with the TF/TFP version used in Acme.

@ddmbr what do you think?

**Comment by ddmbr** [comment]
Hi! I'll try to make a release soon. Which version of TF and Reverb do you need?

**Comment by ethanluoyc** [comment]
Thank you! I think maybe targeting TF 2.12.0 would be good (this is the version of TF that is used by the latest reverb update).

**Comment by hartikainen** [comment]
Yeah `tensorflow==2.12` or `tensorflow==2.13` wheel would be much appreciated. If possible, for `python==3.10`.

**Comment by hartikainen** [comment]
`reverb` just [released v0.12.0](https://pypi.org/project/dm-reverb/0.12.0/) which [supports `tensorflow==2.13`](https://github.com/deepmind/reverb/commit/14cf1831110ab7db4a2fc494335a0e467beb0253#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R95). If the latest launchpad release is still in progress, it would be really neat to get it published with `tensorflow==2.13` 🙂

**Comment by ddmbr** [comment]
Ran into issues, still trying...

**Comment by ethanluoyc** [comment]
Would you like to take a look at https://github.com/ethanluoyc/launchpad/tree/v0.5.2rc0+1 ? This is a working build for TF 2.12. I can create PR if that helps with importing the code for doing things internally.

**Comment by ddmbr** [comment]
Thank you! I'll try your fix later
